### DISCUSSION TITLE
Two-step authentication not working for Mastodon server

### DISCUSSION THREAD

**Discussion by smoothjazzandmore** [mastodon/mastodon#23030] [General]
I've had my account set up for security reasons and now I have no access and locked out, Does anyone have the same issue?

**Comment by rbairwell** [comment]
I haven't, but it might be down to your server (or phone) not having an accurate time setting. Which server (aka instance) are you on or are you administrating the server yourself (if so, how have you installed Mastodon)?

**Reply by smoothjazzandmore** [reply]
masto.ai, I contacted stux, the admin for the server, and he turned off Two-step authentication for me.  Still, if anyone is trying to use it, they may experience problems like I did.
